packages/react-ai/src/components/AIConversation/createProvider.tsx

```diff
@@ -16,6 +20,14 @@ export default function createProvider<
   }: {
     children?: React.ReactNode;
   }): React.JSX.Element {
-    return <ElementsProvider elements={elements}>{children}</ElementsProvider>;
+    return (
+      <ElementsProvider elements={elements}>
+        <AvatarsProvider avatars={avatars}>
+          <ActionsProvider actions={actions}>
+            <MessagesProvider messages={messages}>{children}</MessagesProvider>
```

Comment:
note: totally fine for now since we're using mock data but when we integration with conversation api/hook I will change this so that `messages` is passed in to the provider as a prop

Reply:
Will probably also update this to include the send message api call